nautobot_golden_config/filters.py

```diff
@@ -365,24 +230,24 @@ class Meta:
         """Boilerplate filter Meta data for compliance rule."""
 
         model = models.ComplianceRule
-        fields = ["q", "platform", "feature"]
+        fields = ["q", "platform", "feature", "id"]
```

Comment:
Similarly, since `q` is an explicit CharFilter on this class, you can omit it from `fields` as well.